domains/client/domain-executor/src/utils.rs

```diff
@@ -119,6 +121,22 @@ pub(crate) fn shuffle_extrinsics<Extrinsic: Debug>(
     shuffled_extrinsics
 }
 
+pub(crate) fn read_core_domain_runtime_blob(
+    system_domain_bundle: &[u8],
+    core_domain_id: DomainId,
+) -> Result<Vec<u8>, WasmError> {
+    let system_runtime_blob = RuntimeBlob::new(system_domain_bundle)?;
+
+    let section_contents_name = core_domain_id.link_section_name();
+    let embedded_runtime_blob = system_runtime_blob
+        .custom_section_contents(&section_contents_name)
```

Comment:
Is this something RuntimeBlob provides for a given section name ?
Interesting

Reply:
We are adding `#[link_section = "..."]` into the code, so when WASM is compiled, the contents of the static will be available for direct access without the need to instantiate a VM and call functions on it.

Reply:
Wasm has custom sections in which you can add some data as a payload, substrate uses it to store the runtime version so that you can access the runtime version effectively without spinning up a VM.